libsolidity/Types.cpp

```diff
@@ -284,7 +282,15 @@ IntegerConstantType::IntegerConstantType(Literal const& _literal)
 
 bool IntegerConstantType::isImplicitlyConvertibleTo(Type const& _convertTo) const
 {
-	TypePointer integerType = getIntegerType();
+	auto integerType = getIntegerType();
+	if (_convertTo.getCategory() == Category::FixedBytes)
+	{
+		FixedBytesType const& convertTo = dynamic_cast<FixedBytesType const&>(_convertTo);
```

Comment:
It's easier than typing TypePointer :)
Will change it back. I believe that line must have gone through many changes in the commits of the PR and thus missed this detail